storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -6715,6 +6776,50 @@ static void xtrabackup_prepare_func(int argc, char **argv) {
 
   Tablespace_map::instance().deserialize("./");
 
+  /* Handle `RENAME/DELETE` DDL files produced by DDL tracking during backup */
+  err = xb_data_files_init();
```

Comment:
Add a comment here, we are processing .ibd.ren and .ibd.del after we load tablespaces. this is because
we find the tablespace to be deleted or renamed by space_id.